src/demo.cmd.js

```diff
@@ -100,12 +103,24 @@ class InitCommand {
     }
 
     // #181 cover edge case: make sure git is properly configured
-    if (!await fse.pathExists(`${os.homedir()}/.gitconfig`)) {
-      throw new Error(`
-It seems like Git has not yet been setup on this system. 
-
-See https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup for more information.  
-`);
+    try {
+      if (await this.execAsync('git --version') === 0) {
+        if (!await fse.pathExists(`${os.homedir()}/.gitconfig`)) {
```

Comment:
You should use `path.resolve` instead of string concatenation. This might also help with the Windows issues.